fe1-web/src/core/components/Button.tsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { StyleSheet, View, ViewStyle } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+import { Border, Colors, Spacing } from '../styles';
+
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    marginBottom: Spacing.x1,
+  } as ViewStyle,
+  button: {
+    padding: Spacing.x05,
+
+    borderColor: Colors.accent,
+    backgroundColor: Colors.accent,
+    borderWidth: Border.width,
+    borderRadius: Border.radius,
+  } as ViewStyle,
+  negative: {
+    borderColor: Colors.contrast,
+  } as ViewStyle,
+});
+
+const Button = (props: IPropTypes) => {
```

Comment:
Hmm that would be an option but then people looking for a button will just probably just use the first thing the IDE will suggest when writing Button (which will then be the react-native button).

But since the result will look different and it can be caught in a code review I think your suggestion makes sense!

Reply:
Yeah I get it, but then it can be confusing since 2 components will have the same name, and it will depend on where it has been imported, so renaming it is nice, thanks!